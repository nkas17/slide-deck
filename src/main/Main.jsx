import React from 'react';
import { hot } from 'react-hot-loader';
import IntroContainer from '../pages/intro/IntroContainer';
import './main.css';

// eslint-disable-next-line react/prefer-stateless-function
class Main extends React.PureComponent {
	render() {
		return (
			<IntroContainer />
		);
	}
}

export default hot(module)(Main);
