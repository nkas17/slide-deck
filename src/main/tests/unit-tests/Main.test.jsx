import React from 'react';
import renderer from 'react-test-renderer';
import Main from '../../Main';

describe('Main', () => {
	it('renders correctly', () => {
		const tree = renderer
			.create(<Main />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
