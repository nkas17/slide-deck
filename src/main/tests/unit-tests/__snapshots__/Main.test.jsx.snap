// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<main>
  <div
    className="row ao-top-32"
  >
    <div
      className="column"
    >
      <div
        className="ao-contentbox"
      >
        <img
          alt="react"
          className="ao-float-left ao-right-48"
          height="180"
          src="./assets/react.png"
          width="200"
        />
        <h1
          className="page-intro ao-top-32"
        >
          Hello React
        </h1>
      </div>
    </div>
  </div>
</main>
`;
